### ISSUE TITLE
Remove references to w3c-test.org

### ISSUE THREAD

**Issue by edent** [web-platform-tests/wpt#40483]
And not just for me - https://downforeveryoneorjustme.com/w3c-test.org?proto=http

**Comment by gsnedders** [comment]
You probably want http://wpt.live/, which I thought w3c-test.org had redirected to for years. I forget who controlled the W3C domain? Maybe @sideshowbarker knows?

**Comment by sideshowbarker** [comment]
Yeah, I control the server running on w3c-test.org, and it’s not intentionally not being maintained any longer. I asked the W3C systems team some months ago to completely pull the plug on the underlying VM, and my understanding from them is that they’d schedule time to do that. So it seems they now may have done it.

**Comment by edent** [comment]
Fair enough. Looks like it is still referenced in a few places:
https://github.com/search?q=repo%3Aweb-platform-tests%2Fwpt%20w3c-test&type=code

**Comment by sideshowbarker** [comment]
> Fair enough. Looks like it is still referenced in a few places: [github.com/search?q=repo%3Aweb-platform-tests%2Fwpt%20w3c-test&type=code](https://github.com/search?q=repo%3Aweb-platform-tests%2Fwpt%20w3c-test&type=code)

Yup, we should absolutely fix those

**Comment by Delta456** [comment]
I just saw http://wpt.live/ isn't working as well.
